### PR TITLE
Utm visit 347

### LINKED ISSUE #347: Reveal page visit stats (utm)

### PR DESCRIPTION
**Pull request by JeffAbrahamson**
Add some basic visitor stats for admins.
Closes #347.

### CODE REVIEW

File: transport_nantes/utm/templatetags/utm.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(days=7))).\
+        count()
+
+    context['robots'] = robots.count()
+    context['humans'] = humans.count()
+    context['humans_unique'] = humans_unique.count()
+
+    sources = UTM.objects.filter(base_url=path, ua_is_bot=False) \
+                         .values('source').distinct()
+    print(sources)
```
Comment (Shriukan33) [inline transport_nantes/utm/templatetags/utm.py]: Dev print left here

File: transport_nantes/utm/templatetags/utm.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    context['sources'] = {}
+    for source in sources:
+        this_source = source['source']
+        print('source', this_source)
```
Comment (Shriukan33) [inline transport_nantes/utm/templatetags/utm.py]: Print left here, you can use the logger if this is relevant (the logger will also print in console while in dev)

**Review by Shriukan33 — COMMENTED** [review]
Some prints left, maybe add logging if this is relevant ? 
Otherwise it seems relevant and not too intrusive.

File: transport_nantes/utm/templatetags/utm.py (hunk 1 above)
Comment (JeffAbrahamson) [reply transport_nantes/utm/templatetags/utm.py]: Hmm, I'm at 9d14877, I don't see this...

File: transport_nantes/utm/templatetags/utm.py (hunk 2 above)
Comment (JeffAbrahamson) [reply transport_nantes/utm/templatetags/utm.py]: I don't see this at 9d14877

Comment (JeffAbrahamson) [reply transport_nantes/utm/templatetags/utm.py]: Ah, I see, I could fix them in `HEAD^`.  Just a sec.

**Comment by JeffAbrahamson** [comment]
pushed fix, no change at HEAD, only history.

**Comment by Shriukan33** [comment]
LGTM

**Pull request merged**